libs/analytics/src/lib/components/map/map.component.html

```diff
@@ -1,49 +1,71 @@
 <header>
-  <article>
-    <h5>
-      <mat-icon svgIcon="landscape"></mat-icon>
-      <span>Split by Company Territory</span>
-    </h5>
-    <p class="mat-caption">Learn where are the interested people from.</p>
-  </article>
+  <ng-content></ng-content>
   <button mat-icon-button *ngIf="selected" (click)="toggleSelect(selected)" matTooltip="Remove selection">
     <mat-icon svgIcon="refresh_filters"></mat-icon>
   </button>
 </header>
 
-<ng-container *ngIf="!isLoading else loading">
-  <world-map *ngIf="top3.length; else empty">
+<section *ngIf="!isLoading else loading" [ngClass]="{'map-wrapper':horizontal}">
```

Comment:
For consistency, I wouldn't call this class `map-wrapper` but also `horizontal`
```suggestion
<section *ngIf="!isLoading else loading" [ngClass]="{horizontal}">
```